### PR TITLE
Better usage synopses for subcommands

### PR DESCRIPTION
**Pull request by edsantiago**
Conceptually simple: include, where applicable, a brief
description of command-line options for each subcommand.

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
VERY LOW PRIORITY. And I realize of course that I'll need to rebase after (if) #2452 is merged.

I also realize it'll be tedious to review. I'm sorry. Possible suggestions:
```
$ git log -1 -p | egrep '^\+\s' |less
[shows a brief summary of each added line]

$ egrep -r 'Use: *"[a-z]+"' cmd/podman
[shows the ones I omitted or missed.]
FIXME: I'd like suggestions for 'trust'
```

@baude PTAL but again very low priority

**Comment by edsantiago** [comment]
CI failure: gating: because I added `DisableFlagsInUseLine` (to avoid a misleading addition of "[flags]" to the use string). Someone isn't happy with the indentation. Should I update that in `init` instead?

File: cmd/podman/exists.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ var (
 	}
 
 	_containerExistsCommand = &cobra.Command{
-		Use:   "exists",
+		Use:   "exists CONTAINER",
 		Short: "Check if a container exists in local storage",
 		Long:  containerExistsDescription,
 		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (baude) [inline cmd/podman/exists.go]: would you mind adding examples for all the exists commands?

Comment (edsantiago) [reply cmd/podman/exists.go]: Not at all; I'll work on it tomorrow AM. Thanks for the suggestion.

**Comment by baude** [comment]
@edsantiago i think this looks great... given it is outward facing, would like thumb up from @rhatdan and @mheon . i pulled the PR and really like how it looks.  Adding example for the fellas here:

```
sudo podman attach --help
The podman attach command allows you to attach to a running container using the container's ID or name, either to view its ongoing output or to control it interactively.

Usage:
  podman attach [flags] CONTAINER
...
sudo podman create --help
Creates a new container from the given image or storage and prepares it for running the specified command. The container ID is then printed to stdout. You can then start it at any time with the podman start <container_id> command. The container will be created with the initial state 'created'.

Usage:
  podman create [flags] IMAGE [COMMAND [ARG...]]

```

**Pull request merged**